spec/requests/home_spec.rb

```diff
@@ -9,13 +9,14 @@
         sign_in user
         get root_path
         expect(response).to have_http_status(:success)
+        expect(response.body).to include('Last Activities')
```

Comment:
you should use I18n here :)

Reply:
Interesting, I was not aware that we should make the tests international as well. Done via e19737e247b450b00059b2c9801d6a78e0f14ab7 :-)